### DISCUSSION TITLE
Setting up Oh My Posh on Windows 11, Powershell 7.3.4, and Terminal Preview 1.14.1450.0

### DISCUSSION THREAD

**Discussion by jmakumbi** [JanDeDobbeleer/oh-my-posh#2415] [Troubleshoot] [answered]
Hello,
I am trying to get Oh-my-posh working on Windows 11, Powershell 7.3.4, and Terminal Preview 1.14.1450.0

![image](https://user-images.githubusercontent.com/24497595/174302527-0980b071-6fbf-4ea0-bd91-a589c4cf5cc4.png)

Above is what I am getting.
I have checked the fonts and even tried uninstalling (yuck) and reinstalling it.
May I please have a walkthrough on how to get everything working again?
I originally used Hanselman's blog for guidance, but things have moved on since so a lot of the stuff is outdated.

**Comment by lewis-yeung** [comment, marked as answer]
Seems that you should change the font in Windows Terminal settings. Set it to the Nerd Font that you have installed.

![screenshot](https://user-images.githubusercontent.com/83903009/174315040-cae4c1a6-1f58-4a3d-b383-c5efa5334fe2.png)

Ref: [Fonts | Oh My Posh](https://ohmyposh.dev/docs/installation/fonts#windows-terminal)

**Reply by jmakumbi** [reply]
Thanks, the mistake was that the settings powershell were overriding the defaults.
